lib/services/access_request.go

```diff
@@ -1176,12 +1181,8 @@ func (m *RequestValidator) Validate(ctx context.Context, req types.AccessRequest
 
 		now := m.clock.Now().UTC()
 
-		// Calculate the expiration time of the Access Request (how long it
-		// will await approval).
-		ttl, err := m.requestTTL(ctx, identity, req)
-		if err != nil {
-			return trace.Wrap(err)
-		}
+		// Default to the regular request TTL.
+		ttl := requestTTL
```

Comment:
Can we eliminate this temporary variable and just inline `requestTTL` below?

Reply:
So I actually kept that variable to maintain this behavior later in the function:
```
		// If the maxDuration flag is set, consider it instead of only using the session TTL.
		if maxDuration > 0 {
			req.SetSessionTLL(now.Add(minDuration(sessionTTL, maxDuration)))
			ttl = maxDuration
		} else {
			req.SetSessionTLL(now.Add(sessionTTL))
			ttl = sessionTTL
		}
```

Reply:
Ah, makes sense 👍